cafebabel/static/js/script.js

```diff
@@ -26,6 +26,40 @@ Array.from(document.querySelectorAll('a')).forEach(a => {
     a.setAttribute('target', '_blank')
 })
 
+/* smooth scroll to anchor https://stackoverflow.com/a/17733311/6481285 */
+function animate(elem, style, from, to, time) {
+  if (!elem) {
+    return
+  }
+  var start = new Date().getTime(),
+    timer = setInterval(() => {
+      var step = Math.min(1, (new Date().getTime() - start) / time)
+      elem[style] = from + step * (to - from)
+      if (step === 1) {
+        clearInterval(timer)
+      }
+    }, 25)
```

Comment:
`/* Delay in ms, set to 25 because… */`

Reply:
I removed it. Don't know why it was here